tests/ctst/steps/pra.ts

```diff
@@ -163,7 +164,7 @@ async function waitForPhase(
     return false;
 }
 
-const installTimeout = 360000;
+const installTimeout = 600000;
```

Comment:
why is it increasing? is it really the issue here?

Reply:
I got the same issue on my build, reviewing the logs showed no error, it was just slower than 6min

Reply:
> I am testing some things for the flaky tests, this is not related to this ticket exactly, just trying to make things less flaky..

please do that in another branch (even if based on top of this one) : try to keep things separate, and one ticket = one PR

Reply:
ok